.github/workflows/pull-request.yml

```
@@ -414,4 +426,4 @@
           exit 1
       - name: Cleanup - Free ports by stopping containers
         if: always()
-        run: docker compose down
+        run: docker compose down
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)